buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestIntegTestTask.groovy

```diff
@@ -18,30 +18,36 @@
  */
 package org.elasticsearch.gradle.test
 
-import com.carrotsearch.gradle.junit4.RandomizedTestingTask
 import org.elasticsearch.gradle.VersionProperties
 import org.elasticsearch.gradle.testclusters.ElasticsearchCluster
 import org.elasticsearch.gradle.testclusters.TestClustersPlugin
 import org.gradle.api.DefaultTask
 import org.gradle.api.Task
 import org.gradle.api.execution.TaskExecutionAdapter
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
 import org.gradle.api.tasks.Copy
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskState
 import org.gradle.api.tasks.options.Option
+import org.gradle.api.tasks.testing.Test
 import org.gradle.plugins.ide.idea.IdeaPlugin
+import org.gradle.process.CommandLineArgumentProvider
 
 import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import java.util.stream.Stream
+
 /**
  * A wrapper task around setting up a cluster and running rest tests.
  */
-public class RestIntegTestTask extends DefaultTask {
+class RestIntegTestTask extends DefaultTask {
+
+    private static Logger LOGGER = Logging.getLogger(RestIntegTestTask)
```

Comment:
the convention in existing code is  to use lowercase for these `logger`

Reply:
This should be marked `final` actually which would make the current case appropriate I believe.

Reply:
This is a matter of personal preference for sure, for me uppercase logger hurts my eyes and fingers, final or not :)